hpe3parclient/client.py

```diff
@@ -491,7 +491,7 @@ def createVolume(self, name, cpgName, sizeMiB, optional=None):
             - EXISTENT_SV - Volume Exists already
 
         """
-        info = {'name': name, 'cpg': cpgName, 'sizeMiB': sizeMiB}
+        info = {'name': name, 'cpg': cpgName, 'sizeMiB': sizeMiB, 'objectKeyValues': [{'key': 'type', 'value': self.app_type}]}
```

Comment:
Ideally the objectKeyValues map should come as part of optional arguments to the createVolume call. Use this approach from the caller library.

Reply:
python-3parclient should'nt be changed for this kind of changes.

Reply:
Please do NOT make the default client debug level ON!!!!
It's a major security risk as well to enable SSL warnings by default.  Yes, they are annoying, but they are there for a reason.   Always always always error on the side of security.